### DISCUSSION TITLE
pake 在 mac 端 如何打包 windows 应用啊

### DISCUSSION THREAD

**Discussion by hlf2016** [tw93/Pake#119] [Q&A]
如题 pake 在 mac 端 如何打包 windows 应用啊  
在[这里](https://github.com/tw93/Pake/blob/master/bin/README.md)没找到相关命令行参数啊  好像只能打包 mac

**Comment by hlf2016** [comment]
咋没人理啊 是我问的方式不对吗

**Reply by cnmeeia** [reply]
你可以用 GitHub action 打包  三个系统的app

**Reply by pyy-alt** [reply]
怎么打？

**Comment by pan93412** [comment]
之後會開發 Windows 的打包方式～

**Comment by keventseng** [comment]
这个现在有解决方法了吗？

**Reply by kidylee** [reply]
fork一份到自己的账号下面，然后用github action打包下载就可以了
